app/main.py

```diff
@@ -1,8 +1,24 @@
+from typing import List
+
+
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a class-level dictionary `people` to store instances of `Person` can lead to issues if the class is used in a multi-threaded environment or if multiple datasets are processed simultaneously. Consider using instance-level attributes or another approach to manage relationships.